packages/core/src/ui/UIGroup.ts

```
@@ -0,0 +1,208 @@
+      }
+    }
+    if (flags & GroupModifyFlags.RaycastEnable) {
+      const raycastEnable =
+        this._raycastEnabled &&
+        (!this._ignoreParentGroup && parentGroup ? parentGroup?._getGlobalRaycastEnable() : true);
+      if (this._globalRaycastEnable !== raycastEnable) {
+        this._globalRaycastEnable = raycastEnable;
+        passDownFlags |= GroupModifyFlags.RaycastEnable;
+      }
+    }
+    if (!!flags) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)